app/categories/views.py

```diff
@@ -0,0 +1,147 @@
+from flask.sansio.blueprints import Blueprint
+from flask.views import View
+from flask import render_template, redirect, url_for, session, request
+from app.categories.forms import QuestionForm
+from app.session import set_category_question_answer, get_category_question_answer
+
+
+class CategoryPage(View):
+    template: str = None
+
+    def __init__(self, template, *args, **kwargs):
+        self.template = template
+
+    def dispatch_request(self):
+        return render_template(self.template)
+
+
+class IndexPage(CategoryPage):
+    def dispatch_request(self):
+        session.clear()
+        return render_template(self.template)
+
+
+class CategoryLandingPage(CategoryPage):
+    init_every_request = False
+
+    question_title: str = ""
+
+    category: str = ""
+
+    routing_map: dict[str, str] = {}
+
+    def __init__(self, blueprint: Blueprint):
+        super().__init__(self.template)
+        for answer, next_page in self.routing_map.items():
```

Comment:
Can this adding of routes be made more explicit? Maybe called from urls.py as a dedicated method?

Reply:
Yes, good point!